sale_layout_multi_company/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Lorenzo Battistini - Agile Business Group
+# Copyright 2018 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sale Layout - Multi company",
+    "summary": "Multi company features for sale_layout",
+    "version": "10.0.1.0.0",
+    "category": "Sales Management",
+    "website": "https://github.com/OCA/multi-company/tree/10.0/"
```

Comment:
just `https://github.com/OCA/multi-company`

Reply:
I added the module's complete address because I think it's nice for the user to be redirected directly to the module's page; it shuìould be ok according to https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L9